tests/models/superpoint/test_modeling_superpoint.py

```diff
@@ -260,7 +260,7 @@ def test_inference(self):
         inputs = preprocessor(images=images, return_tensors="pt").to(torch_device)
         with torch.no_grad():
             outputs = model(**inputs)
-        expected_number_keypoints_image0 = 567
```

Comment:
Wondering why the number of keypoints changed :)

Reply:
,